### PR TITLE
dditional contextifying for twitter

### PR DESCRIPTION
**Pull request by maxtaco**
- was forgotten in the previous pass through proofs

### CODE REVIEW

File: go/libkb/proof_support_twitter.go [hunk 1]
```diff
@@ -79,7 +82,7 @@ func (rc *TwitterChecker) findSigInTweet(h SigHint, s *goquery.Selection) ProofE
 }
 
 func (rc *TwitterChecker) CheckStatus(g *GlobalContext, h SigHint) ProofError {
-	res, err := G.XAPI.GetHTML(APIArg{
+	res, err := g.XAPI.GetHTML(APIArg{
```
Comment (patrickxb) [inline go/libkb/proof_support_twitter.go]: can you contextify APIArg too?

Comment (maxtaco) [inline go/libkb/proof_support_twitter.go]: looks like i forgot to in my pass, i have to fix a bunch then.

Comment (patrickxb) [inline go/libkb/proof_support_twitter.go]: there's a constructor, NewAPIArg

**Comment by patrickxb** [comment]
LGTM

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 470d5c41c7b1...c32df9942e62 introduced some visual changes on linux:
 * Changed: **unlock_folders-Single** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470078795239-470d5c41c7b1-c32df9942e62-linux/unlock_folders-Single-470d5c41c7b1.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470078795239-470d5c41c7b1-c32df9942e62-linux/unlock_folders-Single-c32df9942e62.png)  
   unlock_folders-Single comparison

**Pull request merged**